module/behaviour/skills/HeadBehaviourSoccer/src/HeadBehaviourSoccer.cpp

```diff
@@ -316,7 +306,7 @@ namespace behaviour {
                           // If we arent getting up, then we can update the plan if necessary
                           if (updatePlan) {
                               if (lost) {
-                                  lastPlanOrientation = Transform3D(convert(sensors.Htw)).rotation();
+                                  lastPlanOrientation = Eigen::Affine3d(sensors.Htw).rotation();
```

Comment:
Is it better/more efficient/easier to just take the top-left 3x3 corner of `sensors.Htw` here rather than converting to an Affine3d?
```suggestion
                                  lastPlanOrientation = sensors.Htw.topLeftCorner<3, 3>();
```

Reply:
Nah, converting to an Affine3d is almost free. Also while we're here can we convert that variable to `Rtw` so I don't go insane :P